<!DOCTYPE html>
<!--
    Plain-Academic by Vasilios Mavroudis
    Released under the Simplified BSD License/FreeBSD (2-clause) License.
    https://github.com/mavroudisv/plain-academic
-->

<html lang="en">
<head>
  <title>Fabien Plisson</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
</head>
<body>


<!-- Navigation -->
	<nav class="navbar navbar-inverse navbar-static-top" role="navigation">
	  <div class="container">
		<div class="navbar-header">
		  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
		</div>

		<!-- Collect the nav links, forms, and other content for toggling -->
		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		  <ul class="nav navbar-nav">
				  <li><a href="index.html">Home</a></li>
				  <li><a href="research.html">Research</a></li>
				  <li><a href="pubs.html">Publications</a></li>
				  <li><a href="talks.html">Presentations</a></li>
			  	  <li><a href="courses.html">Courses</a></li>
			  	  <li><a href="crew.html">Lab Members</a></li>
			  	  <li><a href="adhoc.html">Ad hoc Activities</a></li>
			          <li><a href="img/CV_FabienPlisson.pdf">CV</a></li>

		  </ul>
		</div>
	  </div>
	</nav>

  <!-- Page Content -->
    <div class="container">

        <div class="row">

	    <!-- Publications -->
            <div class="col-md-8" style="min-height: 100vh; height: auto;">
                <h2 id="publications">Contributions</h2>
		<p>
		<strong>Preprints or Under Review</strong><br/>
                <ol reversed>

		<li class="paper" words="Multistep organic semisynthesis; natural products; anticancer activity; ubiquitin-proteasome">
                    Derguini, F.; Marcadet, F., <strong>Plisson, F.</strong>; Rahmani, R.; Lavigne, F.; Menendez, C.; Long, c.; Marcourt, L.; & Massiot, G.<sup>*</sup>. Tagitinin C and derivatives, as proteasome inhibitors. 2024 [<a href="to be announced"></a>]
		</li>
			
		<li class="paper" words="Algorithmic bias; Antibiotics; Antimicrobial peptides; Antimicrobial resistance; Generative modeling; Machine learning; Predictive modeling; Representation learning">
                    Aguilera-Puga, M. d. C. & <strong>Plisson F.</strong><sup>*</sup>. Structure-aware machine learning strategies for antimicrobial peptide discovery. 2024/02 [<a href="https://doi.org/10.21203/rs.3.rs-3938402/v1">Research Square</a>] [<a href="https://github.com/Puga8Ma/Structure-aware-ML-for-AMP-discovery">Github</a>]
		</li>
		</ol>
		<br><br>

                <strong>Book Chapters</strong><br/>
                <ol reversed>
                <li class="paper" words="Algorithmic bias; Antibiotics; Antimicrobial peptides; Antimicrobial resistance; Generative modeling; Machine learning; Predictive modeling; Representation learning">
                    Aguilera-Puga, M. d. C.; Cancelarich, N. L.; Marani, M. M.; de la Fuente-Nuñez, C.<sup>*</sup> & <strong>Plisson F.</strong><sup>*</sup>. Accelerating the
		    discovery and design of antimicrobial peptides with artificial intelligence. <i>Methods in Molecular Biology</i> 2024; 2714 : 329-352. [<a href="https://doi.org/10.1007/978-1-0716-3441-7_18">DOI</a>]
		</li>
		</ol>
           	<br><br>

                <strong>Journal Articles</strong><br/>
		<ol reversed>
    <li class="paper" words="Algorithmic bias; Antibiotics; Antimicrobial peptides; Antimicrobial resistance; Generative modeling; Machine learning; Predictive modeling; Representation learning">
                      Aguilera-Puga, M. d. C. & <strong>Plisson F.</strong><sup>*</sup>. Structure-aware machine learning strategies for antimicrobial peptide discovery. 2024; 14 : 11995. [<a href="https://doi.org/10.1038/s41598-024-62419-y">Scientific Reports</a>] [<a href="https://doi.org/10.21203/rs.3.rs-3938402/v1">Research Square</a>] [<a href="https://github.com/Puga8Ma/Structure-aware-ML-for-AMP-discovery">Github</a>]<br>
                      <strong> Our study is the first assessment of structural bias in model datasets and of the structural effects upon machine learning-guided peptide bioactivity prediction. </strong>
  	</li>
    &thinsp;
		<li class="paper" words="Antibiotics; Antimicrobial peptides; Bacterial resistance; Veterinary medicine; Food Security; bovine mastitis">
                    Robles-Ramirez, O.; Osuna, J. G.; <strong>Plisson F.</strong>; Barrientos-Salcedo; C.<sup>*</sup>. Antimicrobial Peptides in Livestock: A Review with a One Health Approach.
		    <i>Frontiers in Cellular and Infection Microbiology</i> 2024; vol. 14. [<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1339285/abstract">DOI</a>]
		</li>
		&thinsp;
    <li class="paper" words="Algorithmic bias; Antibiotics; Antimicrobial peptides; Structure-activity landscape; Post-translational modifications; Representation learning">
                    López-López, E.; Robles, O.; <strong>Plisson F.</strong>; Medina-Franco, J. L.<sup>*</sup>. Mapping the structure-activity landscape of non-canonical peptides with MAP4 fingerprinting.
		   <i>Digital Discovery</i> 2023, 2, 1494-1505. [<a href="https://doi.org/10.1039/D3DD00098B">DOI</a>] [<a href="https://doi.org/10.26434/chemrxiv-2023-0j38z">ChemXriv</a>]
		</li>
		&thinsp;
		<li class="paper" words="Algorithmic bias; Antibiotics; Antimicrobial peptides; Structure-activity landscape; Post-translational modifications; Representation learning">
                    Aldas-Bulos, V. D. & <strong>Plisson F.</strong><sup>*</sup>. Benchmarking protein structure predictors to assist machine learning-guided peptide discovery.
		   <i>Digital Discovery</i> 2023, 2, 981-993. [<a href="https://doi.org/10.1039/D3DD00045A">DOI</a>] [<a href="https://doi.org/10.26434/chemrxiv-2023-krc22">ChemXriv</a>] [<a href="https://github.com/DanielAldas/Benchmark-PSP">Github 1</a>] [<a href="https://github.com/plissonf/GRAMPA_structural_landscape">Github 2</a>] <br>
		   <strong> A fast and robust method to assess structural bias in medium-large protein datasets before machine learning modeling. </strong>
		</li>
		&thinsp;
		<li class="paper" words= "Anticancer peptides; data scarcity; membranolysis; lipidomics">
                    Martínez-Hernández, C.; Aguilera-Puga, M. d. C. & <strong>Plisson F.</strong><sup>*</sup>. Deconstructing the potency and cell-line selectivity of membranolytic anticancer peptides.
		   <i>ChemBioChem</i> 2023, 24(14):e202300058. [<a href="https://doi.org/10.1002/cbic.202300058">DOI</a>] [<a href="https://doi.org/10.26434/chemrxiv-2023-mdm1x">ChemXriv</a>] <br>
		   <strong> Membrane-active anticancer peptides can respond similarly (potency and selectivity) towards cell lines from different tissues. </strong>
		</li>
		&thinsp;
		<li class="paper" words="Pesticides; soil microbiology">
                   Dennis, P. G.<sup>*</sup>; Kukulies, T.; Forstner, C.; <strong>Plisson, F.</strong>; Eaglesham, G.; Pattison, A. B. The Effects of Atrazine, Diuron, Fluazifop-P-butyl, Haloxyfop-P-methyl,
		   and Pendimethalin on Soil Microbial Activity and Diversity. <i>Applied Microbiology</i> 2023, 3, 79-89. [<a href="https://doi.org/10.3390/applmicrobiol3010007">DOI</a>]
		</li>
		&thinsp;
		<li class="paper" words="Antimicrobial peptides; machine learning">
                   <strong>Plisson, F.</strong> Overcoming the Challenges in Machine Learning-Guided Antimicrobial Peptide Design.
		   <i>Proceedings of the 36th European and the 12th International Peptide Symposium</i> 2022, 207-210. ISBN: 9798987214008. [<a href="http://www.kvinzo.com/36EPS/36EPS.pdf">PDF</a>]
		</li>
		&thinsp;
		<li class="paper" words="Toxins; Venom; Hymenoptera">
                   Guido-Patiño, J. C. & <strong>Plisson, F.</strong><sup>*</sup>. Profiling hymenopteran venom toxins: protein families, structural landscape, biological activities, and pharmacological benefits.
		   <i>Toxicon:X</i> 2022, 14, 100119. [<a href="https://doi.org/10.1016/j.toxcx.2022.100119">DOI</a>]
		</li>
		&thinsp;
		<li class="paper" words="Toxins; Venom; Hymenoptera">
                   Bajorath, J.; Chávez-Hernández, A. L.;  Duran-Frigola, M.; Fernández-de Gortari, E.; Gasteiger, J.; López-López, E.; Maggiora, G. M.; Medina-Franco, J. L.<sup>*</sup>; Méndez-Lucio, O.; Mestres, J.;
		   Miranda-Quintana, R. A.; Oprea, T. I.; <strong>Plisson, F.</strong>; Prieto-Martínez, F. D.; Rodríguez-Pérez, R.; Rondón-Villarreal, P.; Saldívar-Gonzalez, F. I.; Sánchez-Cruz, N.; Valli, M.
		   Chemoinformatics and artificial intelligence colloquium: progress and challenges to develop bioactive compounds.
		   <i>Journal of Chemoinformatics</i> 2022, 14(1), 82. [<a href="https://doi.org/10.1186/s13321-022-00661-0">DOI</a>]
		</li>
		&thinsp;
		<li class="paper" words="Peptides; toxicity; machine learning">
                   Robles-Loaiza, A. A.; Pinos-Tamayo, E. A.; Mendes, B.; Ortega-Pila, J. A.; Proaño-Bolaños, C.; <strong>Plisson, F.</strong>; Teixeira, C.; Gomes, P.; Almeida, J. R.<sup>*</sup>.
		   Profiling hymenopteran venom toxins: protein families, structural landscape, biological activities, and pharmacological benefits. Traditional and Computational Screening of Non-Toxic Peptides and
		   Approaches to Improving Selectivity. <i>Pharmaceuticals</i> 2022, 15, 323. [<a href="https://doi.org/10.3390/ph15030323">DOI</a>]
		</li>
		&thinsp;
		<li class="paper" words="Natural Product Drug Discovery; Artificial Intelligence; Natural Language Processing">
                   Saldivar-González, F. I.; Aldas-Bulos, V. D.; Medina-Franco, J. L. & <strong>Plisson, F.</strong><sup>*</sup>. Natural product drug discovery in the artificial intelligence era.
		   <i>Chemical Science</i> 2022, 13, 1526-1546. [<a href="https://doi.org/10.1039/D1SC04471K">DOI</a>] <br>
		   <strong> Seminal and the most comprehensive review of artificial intelligence technologies (machine learning and natural language processing) applied to natural product drug discovery and molecular design. </strong>
		</li>
		&thinsp;
		<li class="paper" words="SARS-CoV-2; Genomic surveillance; COVID-19 pandemic">
                   Barona-Gómez, F.<sup>#</sup>; Delaye L.<sup>#</sup>; Díaz-Valenzuela, E.<sup>#</sup>; <strong>Plisson, F.</strong><sup>#</sup>; Cruz-Pérez, A.; Díaz-Sánchez, M.; García-Sepúlveda, C. A.; Sanchez-Flores, A.; Pérez-Abreu, R.; Valencia-Valdespino,
		   F. J.; Vega-Magaña, N.; Muñoz-Valle, J. F.; García-González, O. P.; Bernal-Silva, S.; Comas-García, A.; Cibrián-Jaramillo, A.<sup>*</sup>. Phylogenomics and population genomics of
		   SARS-CoV-2 in Mexico during the pre-vaccination stage reveals variants of interest B.1.1.28.4, B.1.1.222, or B.1.1.519 and the nucleocapsid mutation S194L associated with symptoms.
		   <i>Microbial Genomics</i> 2021, Nov;7(11):000684. PMID: 34846283 [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8743546/">PMCID: PMC8743546</a>] [<a href="https://doi.org/10.1099/mgen.0.000684">DOI</a>] [<a href="https://www.medrxiv.org/content/10.1101/2021.05.18.21256128v2">medRxiv</a>] [<a href="https://github.com/plissonf/Phylogenomics_SARS-CoV-2_Mexico">Github</a>]<br>
		   <strong> Part of the genomic surveillance program of SARS-CoV-2 variants in Mexico. </strong>
		   <sup>#</sup> Co-first authors.
		</li>
		&thinsp;
		<li class="paper" words="Cysteine-rich peptides; molecular grafting">
                   González-Castro, R.; Gómez-Lim, M. A. & <strong>Plisson, F.</strong><sup>*</sup>. Cysteine-rich peptides: hyperstable scaffolds for protein engineering.
		   <i>ChemBioChem</i> 2020, 22(6), 961-973. [<a href="https://doi.org/10.1002/cbic.202000634">DOI</a>] [<a href="https://doi.org/10.1002/cbic.202100083">Cover Feature</a>]
		</li>
		&thinsp;
		<li class="paper" words="toxicity; hemolysis; peptides; machine learning;, applicability domain">
                   <strong>Plisson, F.</strong><sup>*</sup>; Ramírez-Sánchez, O. & Martínez-Hernández, C. Machine learning-guided discovery and design of non-hemolytic peptides.
		   <i>Scientific Reports</i> 2020, 10, 16581. [<a href="https://doi.org/10.1038/s41598-020-73644-6">DOI</a>] [<a href="https://github.com/plissonf/ML-guided-discovery-and-design-of-non-hemolytic-peptides">Github</a>] <br>
		   <strong> The first study to report the domain of applicability of peptide predictive models and its limitations using multivariate outlier detectors. </strong>
		</li>
		&thinsp;
		<li class="paper" words="marine natural products;  blood-brain barrier permeability; machine learning">
                   <strong>Plisson, F.</strong><sup>*</sup> & Piggott, A. M. Predicting blood-brain barrier permeability of marine-derived kinase inhibitors using ensemble classifiers reveals potential hits
		   for neurodegenerative disorders. <i>Marine Drugs</i> 2019, 17, 81. [<a href="https://doi.org/10.3390/md17020081">DOI</a>] [<a href="https://github.com/plissonf/BBB-Models">Github</a>] <br>
		   <strong> One of the earliest studies to develop machine learning predictive models applied to (marine) natural products. </strong>
		</li>
		&thinsp;
		<li class="paper" words="GLP-1 receptor; peptide agonists; type 2 diabetes; lactam constraints; helical propensity">
                   <strong>Plisson, F.</strong><sup>#</sup>; Hill, T. A.<sup>#</sup>; Mitchell, J. M.; Hoang, H. N.; de Araujo, A. D.; Xu, W. J.; Cotterell, A.; Edmonds, D. J.; Stanton, R. V.;
		   Derksen, D. R.; Loria, P. M.; Griffith, D. A.; Price, D. A.; Liras, S. & Fairlie, D. P.<sup>*</sup>. Helix-constraints and amino acid substitution in GLP-1 increase cAMP and insulin secretion
		   but not β-arrestin 2 signalling <i>European Journal of Medicinal Chemistry</i> 2017, 127, 703-714. [<a href="https://doi.org/10.1016/j.ejmech.2016.10.044">DOI</a>] <br>
		   <sup>#</sup> Co-first authors.
		</li>
	        &thinsp;
		<li class="paper" words="GLP-1 receptor; peptide agonists; type 2 diabetes; lactam constraints; helical propensity">
                   Hoang, H. N.; Driver, R.W.; Beyer, R.L.; Hill, T.A.; de Araujo, A.D.; <strong>Plisson, F.</strong>; Harrison, R.S.; Goedecke, L.; Shepherd, N. E. & Fairlie, D. P.<sup>*</sup>.
		   Helix Nucleation by the Smallest Known α-Helix in Water. <i>Angewandte Chemie International Edition</i> 2016, 55(29), 8275-8279. [<a href="https://doi.org/10.1002/anie.201602079">DOI</a>] <br>
		</li>
		&thinsp;
		<li class="paper" words="hymenialdisine; Bromopyrrolo-2-aminoimidazoles; kinase inhibitors">
                   <strong>Plisson, F.</strong>; Prasad P.; Xiao, X.; Piggott, A. M.; Huang, X.-C.; Khalil, Z. & Capon, R. J.<sup>*</sup>. Callyspongisines A-D: bromopyrrole alkaloids from an Australian marine sponge, <i>Callyspongia sp.</i>.
		   <i>Organic & Biomolecular Chemistry</i> 2014, 12, 1579-1584. [<a href="https://doi.org/10.1039/C4OB00091A">DOI</a>] <br>
		</li>
		&thinsp;
		<li class="paper" words="ningalins; alkaloids; kinase inhibitors">
                   <strong>Plisson, F.</strong>; Conte, M.; Khalil, Z.; Huang, X.-C.; Piggott A. M. & Capon, R. J.<sup>*</sup>. Kinase inhibitor scaffolds against neurodegenerative disorders from a southern Australian ascidian, <i>Didemnum sp.</i>.
		   <i>ChemMedChem</i> 2012, 7(6), 980-993. [<a href="https://doi.org/10.1002/cmdc.201200169">DOI</a>] <br>
		   <strong>First example of executing molecular docking experiments to understand the mechanism(s) of action of natural products.</strong>
		</li>
		&thinsp;
		<li class="paper" words="lamellarins; alkaloids; kinase inhibitors">
                   <strong>Plisson, F.</strong>; Huang, X.-C.; Khalil, Z.; Zhang, H. & Capon, R. J.<sup>*</sup>. Lamellarins as substrates and inhibitors of P-glycoprotein-mediated multidrug resistance in a human colon cancer cell line.
		   <i>Chemistry – An Asian Journal</i> 2012, 7(7), 1616-1623. [<a href="https://doi.org/10.1002/asia.201101049">DOI</a>] <br>
		</li>
		&thinsp;
		<li class="paper" words="Bromopyrrolo-2-aminoimidazoles; alkaloids; kinase inhibitors">
                   Zhang, H.; Khalil, Z.; Conte, M. M.; <strong>Plisson, F.</strong> & Capon, R. J.<sup>*</sup>. A search for kinase inhibitors and antibacterial agents: Bromopyrrolo-2-aminoimidazoles from a deep-water Great Australian Bight sponge, <i>Axinella sp.</i>
		   <i>Tetrahedron Letters</i> 2012, 53(29), 3784-3787. [<a href="https://doi.org/10.1016/j.tetlet.2012.05.051">DOI</a>] <br>
		</li>
		&thinsp;
		<li class="paper" words="proteasome inhibitors; plant natural products">
                   Long, C.; Beck, J.; Cantagrel, F.; Marcourt, L.; Vendier, L.; David, B.; <strong>Plisson, F.</strong>; Derguini, F.; Vandenberghe, I.; Aussagues, Y.; Ausseil, F.; Lavaud, C.; Sautel, F. & Massiot, G.<sup>*</sup>.
		   Proteasome inhibitors from <i>Neoboutonia melleri.</i> <i>Journal of Natural Products</i> 2012, 75 (1), 34-47. [<a href="https://doi.org/10.1021/np200441h">DOI</a>] <br>
		</li>
		&thinsp;
		<li class="paper" words="proteasome inhibitors; plant natural products">
                   Beck, J.; Guminski, Y.; Long, C.; Marcourt, L.; Derguini, F.; <strong>Plisson, F.</strong>; Grondin, A.; Vandenberghe, I.; Vispé, S.; Brel, V.; Aussagues, Y.; Ausseil, F.; Arimondo, P. B.; Massiot, G.; Sautel, F. & Cantagrel, F.<sup>*</sup>.
		   Semisynthetic Neoboutomellerone derivatives as ubiquitin-proteasome pathway inhibitors. <i>Bioorganic and Medicinal Chemistry</i> 2012, 20 (2), 819-831. [<a href="https://doi.org/10.1016/j.bmc.2011.11.066">DOI</a>] <br>
		</li>
		&thinsp;
		<li class="paper" words= "plant natural products">
                   Ajala, O. S.; Piggott, A. M.; <strong>Plisson, F.</strong>; Khalil, Z.; Huang, X.-C.; Adesegun, S. A.; Coker, H. A. B. & Capon, R. J.<sup>*</sup>.
		   Ikirydinium A: a new indole alkaloid from the seeds of <i>Hunteria umbellata</i>. <i>Tetrahedron Letters</i> 2011, 52, 7125-7127. [<a href="https://doi.org/10.1016/j.tetlet.2011.10.106">DOI</a>] <br>
		</li>
                </ol>
		<br><br>

		<strong>Patents</strong><br/>
		<ol reversed>
                <li class="paper" words="Tagitinine C; Semi-synthesis; proteasome inhibitors">
                   Derguini, F.; <strong>Plisson, F.</strong>; Massiot G. <i>Preparation of tagitinin C and F derivatives as anti-cancer agents.</i> <br>
		   Patent application <strong>FR 2941697 A1 20100806 2010</strong>. [<a href="https://europepmc.org/article/PAT/FR2941697">Europe PMC</a>] [<a href="https://patents.google.com/patent/FR2941697A1/en">Google Patents</a>]
		</li>
                <!--
                <strong>Current Work</strong><br/>
                <ul>
                <li>

                </li>
                </ul>
                -->
                <br><br><br><br>

            </div>
    <!-- /.container -->

</body>

</html>
