<!DOCTYPE html>
<!--
    Plain-Academic by Vasilios Mavroudis
    Released under the Simplified BSD License/FreeBSD (2-clause) License.
    https://github.com/mavroudisv/plain-academic
-->

<html lang="en">
<head>
  <title>Fabien Plisson</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
</head>
<body>


<!-- Navigation -->
	<nav class="navbar navbar-inverse navbar-static-top" role="navigation">
	  <div class="container">
		<div class="navbar-header">
		  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
		</div>

		<!-- Collect the nav links, forms, and other content for toggling -->
		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		  <ul class="nav navbar-nav">
				  <li><a href="index.html">Home</a></li>
				  <li><a href="research.html">Research</a></li>
				  <li><a href="pubs.html">Publications</a></li>
				  <li><a href="talks.html">Presentations</a></li>
			  	  <li><a href="courses.html">Courses</a></li> 
			  	  <li><a href="crew.html">Lab Members</a></li> 
			  	  <li><a href="adhoc.html">Ad hoc Activities</a></li>
			  
		  </ul>
		</div>
	  </div>
	</nav>
  
  <!-- Page Content -->
    <div class="container">

        <div class="row">

	    <!-- Talks -->
            <div class="col-md-8" style="height">    
            <h2 id="projects">Current Projects</h2>

            <a target="_blank"><strong>Accelerating peptide drug discovery</strong></a><br />
            <p>
            The peptide space is vast, but the regions occupied by biologically active peptides account for a limited set of sequences 
	    folding into privileged structures. The advent of artificial intelligence (AI) in life sciences has led to the creation of 
	    machine learning (ML) models capable of predicting sequence-structure-function relationships (that may not be readily apparent) 
	    and generating new proteins with the desired characteristics. Predictive models infer the relationships between the protein 
	    sequences and their biological functions (e.g., thermostability, microbial inhibition, protein binding affinity). Generative 
	    models learn meaningful representations from sequences or structures to create proteins that resemble their native counterparts. 
	    Successful applications of ML-guided protein design have been documented worldwide for protein binders, antigen-specific monoclonal 
	    antibodies, protein families, and enzymes. 
	    My group devises predictive models and bioinformatic pipelines to effectively screen for peptide sequences from sizeable public 
	    repositories or proteo-transcriptomic datasets. Our efforts presently focus on antimicrobial peptides (AMPs) and venom toxins. Both 
	    groups of bioactive peptides are well-documented in curated databases, private repositories, and in-house datasets. Our current research 
	    efforts focus on (i) discovering novel AMPs from untapped extant species and (ii) engineering novel AMPs with reduced or inexistent 
	    hemolytic activity using evolutionary information. For example. we implemented AI models to identify antifungal peptides from hymenopteran 
	    venoms against Candida albicans or the neurotoxin families from scorpion Centruroides sp. transcriptomes. Finally, we apply an array of 
	    protein structure predictions, protein structural alignment methods, and deep learning architectures to expand the functional annotation 
	    of transcriptomic datasets (e.g. Conus marmoreus). Actual methods (e.g. AlphaFold, RosettaFOLD) give excellent results with well-referenced 
	    protein families (>200 examples, above 70 residues in length) but often fail with smaller peptide sequences.
            </p>  

	    <a target="_blank"><strong>Automating peptide drug design.</strong></a><br />
            <p>
	    An essential step in peptide drug design is finding the trade-off between the extraordinary biological activities and highly desirable pharmacokinetic 
	    properties of peptide candidates while reducing their undesirable side effects (e.g., lack of metabolic stability, lability during storage, poor oral 
	    bioavailability, and unwanted toxicity) – a multi-objective optimization. Conventional approaches operate iteratively through a series of modifications, 
	    followed by their physical or biological evaluations – a Design-Make-Test-Analyse (DMTA) cycle. The peptide space is vast - investigating all functionally 
	    interesting analogues is not practical and is an expensive task. Implementing a computational DMTA cycle speeds up the design process by narrowing the list 
	    of peptide candidates for synthesis and biological evaluations. To minimise the impact that structural factors might have upon biological prediction or sequence 
	    generation, researchers voluntarily select sequences based on structural or evolutionary constraints, which presumably adopt the same tridimensional structure(s). 
	    Our recent efforts24 in discovering novel bioactive peptides also revealed that computational designers have so far capitalised on ML architectures to predict and 
	    generate novel α-helical non-hemolytic AMPs21, 26–30. These findings indicate that current ML models do not generalize well and are unfair towards minor structural 
	    classes, limiting the prospect of intellectual property. 
	    </p>

	    <a target="_blank"><strong>Producing our AI-designed peptides.</strong></a><br />
	    <p>
	    Accessing peptide libraries is crucial to understanding and validating our AI-driven hypotheses about complex sequence-structure-function relationships. We have been 
	    exploring ways to produce our peptides through solid-phase peptide synthesis (SPPS) and protein expression in biological systems. During my postdoctoral stays at the 
	    University of Queensland, I synthesised peptide analogues through manual and automated SPPS. Our current efforts have shifted towards the production of peptides in 
	    biological expression systems; bacteria (Escherichia coli), yeast (Pichia pastori), and tobacco plant (Nicotiana benthamiana) due to the institutional facilities in 
	    place. Both strategies are ideal for the production of a handful of bioactive peptides. However,  SPPS and heterologous expression in biological systems have their 
	    own limitations when it comes to generating large numbers of peptides and proteins.
	    </p>
    

	    <h2 id="projects"> Past Projects</h2>

	    <a target="_blank"><strong>Developing novel potent and constrained GLP-1R peptide agonists</strong></a><br />
            <p>
            </p>

	    <a target="_blank"><strong>Studying the physicochemical properties of bioavailable cyclic peptides and macrocycles </strong></a><br />
            <p>
            </p>
		    
	    <a target="_blank"><strong>Predicting the blood-brain barrier permeability of marine-derived kinase inhibitors</strong></a><br />
            <p>
            </p>

	    <a target="_blank"><strong> Repurposing natural product library and discover novel novel S/T kinase inhibitors</strong></a><br />
            <p>
            </p>

	    <a target="_blank"><strong> Discovering novel S/T kinase inhibitors from marine natural product chemistry </strong></a><br />
            <p>
            </p>

            <a target="_blank"><strong> Hit/Lead optimisation of natural product-derived proteasome inhibitors </strong></a><br />
            <p>
            </p>
	    
	    </div>
           
        </div>


    </div>
    <!-- /.container -->
    
</body>

</html>
